contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -692,6 +692,57 @@ is_json_query(List *name)
 	}
 }
 
+/*
+* Parse T-SQL CONTAINS predicate. Currently only supports 
+* ... CONTAINS(column_name, '<contains_search_condition>') ...
+* This function transform it into a Postgres AST that stands for
+* to_tsvector('fts_contains', column_name) @@ to_tsquery('fts_contains', babelfish_fts_contains_rewrite('<contains_search_condition>'))
+*/
+static Node *
+TsqlExpressionContains(char *colId, Node *search_expr, core_yyscan_t yyscanner)
+{
+	A_Expr *fts;
+	Node *to_tsvector_call, *to_tsquery_call;
+
+	to_tsvector_call = makeToTSVectorFuncCall(colId, yyscanner);
+	to_tsquery_call = makeToTSQueryFuncCall(search_expr);
+	
+	fts = makeA_Expr(AEXPR_OP, list_make1(makeString("@@")), to_tsvector_call, to_tsquery_call, -1);
+
+	return (Node *)fts;
+}
+
+/* Transform column_name into to_tsvector('fts_contains', column_name) */
+static Node *
+makeToTSVectorFuncCall(char *colId, core_yyscan_t yyscanner)
+{
+	Node *col;
+	List *args;
+
+	col = makeColumnRef(colId, NIL, -1, yyscanner);
+
+	args = list_make2(makeStringConst("fts_contains", -1), col);
```

Comment:
same comment as in makeToTSQueryFuncCall()

Reply:
fixed